notarize.js

```diff
@@ -0,0 +1,38 @@
+// See: https://medium.com/@TwitterArchiveEraser/notarize-electron-apps-7a5f988406db
+
+'use strict';
+const fs = require('fs');
+const path = require('path');
+const electronNotarize = require('electron-notarize');
+
+module.exports = async params => {
+  if (process.platform !== 'darwin') {
+    return;
+  }
+
+  // Only notarize the app on the master branch
+  // if (process.env.CIRCLE_BRANCH !== 'master') {
+  //   return;
+  // }
+
+  const packageJson = JSON.parse(fs.readFileSync('./package.json'));
```

Comment:
Why not just:

```suggestion
  const packageJson = require('./package.json');
```

Reply:
🤦‍♂ forgot you can just require json files